### DISCUSSION TITLE
UseCookie helper in Nuxt documentation is depreciated in h3

### DISCUSSION THREAD

**Discussion by Lyokolux** [nuxt/nuxt#16833] [General]
Using Nuxt RC8 and H3, I read:
<img width="581" alt="Bildschirmfoto 2022-09-12 um 15 19 03" src="https://user-images.githubusercontent.com/29046067/189664935-8bb4209c-6673-429b-ba46-bc3a1235a236.png">

and there is nothing about it in the [useCookie section](https://v3.nuxtjs.org/api/composables/use-cookie/).

What is be the recommended way to handle cookie in Nuxt 3 then? A documentation update will be great 🙌

**Comment by CamilleScholtz** [comment]


**Reply by TheAlexLichter** [reply]
What exactly is lacking in your opinion? 
Changes are documented in the release notes (and also in the subsequent package changelogs if applicable).

**Comment by TheAlexLichter** [comment]
The method has been renamed to `getCookies` - thus the deprecation. Docs are up to date @ https://www.jsdocs.io/package/h3#useCookie and the `useCookie` section of the Nuxt3 docs also has an up to date example (see https://v3.nuxtjs.org/api/composables/use-cookie/#handling-cookies-in-api-routes)

```js
export default defineEventHandler(event => {
  // Read counter cookie
  let counter = getCookie(event, 'counter') || 0

  // Increase counter cookie by 1
  setCookie(event, 'counter', ++counter)

  // Send JSON response
  return { counter }
})
```

**But feel free to send in a PR to the docs highlighting the change ☺️**
